sdcm/utils/version_utils.py

```diff
@@ -146,7 +146,11 @@ def parse(version_string: str):
 
         # NOTE: make short scylla version like '5.2' be correct semver string
         _scylla_version_parts = _scylla_version.split('.')
-        if len(_scylla_version_parts) == 2:
+        if '-' in _scylla_version_parts[1]:
```

Comment:
Unit tests fail on the single-element input values such as `2023`:
```
15:01:28  >       if '-' in _scylla_version_parts[1]:
15:01:28  E       IndexError: list index out of range
```

So, need to check that second element does exist.

Reply:
fixed